README.md

```diff
@@ -92,23 +99,10 @@ Anything listed in the `secret_keys` entry of that file will automatically be ma
 
 ## Database
 
-
 Create and set up your development and test databases:
 
     $ rails db:setup
     $ rails db:test:prepare
-
-## Asset pipeline
-
-A few tips when working with asset pipeline:
-
-* Ensure that your asset directory is in the asset paths by running the following in the console:
-
-    > Rails.application.assets.paths
-
-* Find out which file is served for a given asset path by running the following in the console:
-
-    > Rails.application.assets['relative_path/to_asset.ext']
```

Comment:
I removed this section, as it's not a necessary step for either development or test setup.

Reply:
I found this confusing (what is "my asset directory?"), anyway..